libs/designsystem/src/lib/components/list/list.component.html

```diff
@@ -14,7 +14,11 @@
 
   <ng-container
     *ngTemplateOutlet="
-      _isSectionsEnabled ? groupedListTemplate : itemsTemplate;
+      _isSectionsEnabled && _isStandAloneEnabled
+        ? groupedListWithStandAloneTemplate
+        : _isSectionsEnabled || _isStandAloneEnabled
+        ? groupedListTemplate
+        : itemsTemplate;
```

Comment:
I can't always remember the precedence rules for the ternary operator and my brain finds it hard to process?

Can you reduce the cognitive overhead?

(optional): Nested ternary operations is often considered a code smell. We could consider using a [ESLint rule](https://eslint.org/docs/latest/rules/no-nested-ternary) to warn against it. (Should probably be handled in a separate issue)

Reply:
Yeah no problem! :) I will move the logic out of the template and make it more readable.

That's a great idea! I will add it to the ESLint branch :)

Reply:
I ended up refactoring the groupedListTemplate, so it could handle multiple cases, which makes us able to convert the ternary operator back to original with no nesting :)